packages/storybook/src/utils/createVisualRegressionStory.tsx

```diff
@@ -0,0 +1,42 @@
+import { StoryObj } from '@storybook/react';
+import { CSSProperties, PropsWithChildren, ReactElement } from 'react';
+
+type VisualRegressionRenderFn = () => ReactElement<void, any>;
+interface CreateVisualRegressionStoryFn {
+  // eslint-disable-next-line no-unused-vars
```

Comment:
Om een of andere reden struikelt ESlint over dat de `render` parameter niet gebruikt is. Dit is een snelle fix zodat ik een rustig weekend kan hebben. Ik zal hier volgende week wel nog naar kijken. Als iemand een suggestie heeft hoor ik dat graag.

Reply:
Geen oplossing voor nu